browser/ui/brave_shields_data_controller.h

```diff
@@ -78,6 +78,7 @@ class BraveShieldsDataController
   void SetIsNoScriptEnabled(bool is_enabled);
   void SetIsHTTPSEverywhereEnabled(bool is_enabled);
   void AllowScriptsOnce(const std::vector<std::string>& origins);
+  void BlockAllowedScripts(const std::vector<std::string>& origins);
```

Comment:
block allowed scripts? Can you please add a comment explaining what it does because it makes no sense to block an allowed script.

Reply:
I think this should probably be `BlockScriptsOnce` to match `AllowScriptsOnce`?